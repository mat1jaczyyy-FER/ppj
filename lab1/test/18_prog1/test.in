x = 123 // var
za n od 3 do 321
  x = x - 1
az

// komentar
// y+z
za i od -x do x
  x = x*x
// az
