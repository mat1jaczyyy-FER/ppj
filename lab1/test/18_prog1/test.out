IDN 1 x
OP_PRIDRUZI 1 =
BROJ 1 123
KR_ZA 2 za
IDN 2 n
KR_OD 2 od
BROJ 2 3
KR_DO 2 do
BROJ 2 321
IDN 3 x
OP_PRIDRUZI 3 =
IDN 3 x
OP_MINUS 3 -
BROJ 3 1
KR_AZ 4 az
KR_ZA 8 za
IDN 8 i
KR_OD 8 od
OP_MINUS 8 -
IDN 8 x
KR_DO 8 do
IDN 8 x
IDN 9 x
OP_PRIDRUZI 9 =
IDN 9 x
OP_PUTA 9 *
IDN 9 x
